backend-rust/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "concordium-scan"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1"
```

Comment:
I think we should specify full versions for dependencies, [see some argument for why here](https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277).

Reply:
If we know the minimum supported version of these then sure, but I don't see the argument when include the `Cargo.lock` file.
This seems more relevant for libraries.

Reply:
If you add dependencies later on, that could change the versions in the lock file, as far as I know. It is relevant for libraries but it is good practice in general.

Reply:
I don't see why we should include a patch version unless we know this patch version is the minimum supported.
If we add a new dependency we want it to resolve the shared dependencies and I don't think cargo will bump unrelated dependencies.
In case it resolves to a version we don't support, we can then add a patch version with a comment to why this is the case.

Reply:
>I don't see why we should include a patch version unless we know this patch version is the minimum supported.

But right now, if you say `anyhow = "1"` (just as an example), it seems like the application should work and build with anyhow version `1.0.0`, but that may actually not be the case, because we haven't tested that because cargo chose a later version and we've only tested with that version.